Sources/Web3Core/Oracle/GasOracle.swift

```diff
@@ -60,7 +60,7 @@ final public class Oracle {
     ///
     /// - Parameter array: `[[min], [middle], [max]]` 2 dimensional array
     /// - Returns: `[min, middle, max].count == self.percentiles.count`
-    private func soft(twoDimentsion array: [[BigUInt]]) -> [BigUInt] {
+    private func soft(twoDimension array: [[BigUInt]]) -> [BigUInt] {
```

Comment:
Please undo this changes, as they affecting code, rather then comments and should be reviewed separately.